src/avfoundation.cs

```diff
@@ -8442,6 +8475,35 @@ interface AVCaptureDevice {
 		[Export ("activeVideoMaxFrameDuration", ArgumentSemantic.Copy)]
 		CMTime ActiveVideoMaxFrameDuration { get; set; }
 
+		[iOS (10,0), NoMac]
+		[Export ("lockingFocusWithCustomLensPositionSupported")]
+		bool LockingFocusWithCustomLensPositionSupported { [Bind ("isLockingFocusWithCustomLensPositionSupported")] get; }
+
+		[iOS (10,0), NoMac]
+		[Export ("lockingWhiteBalanceWithCustomDeviceGainsSupported")]
+		bool LockingWhiteBalanceWithCustomDeviceGainsSupported { [Bind ("isLockingWhiteBalanceWithCustomDeviceGainsSupported")] get; }
+
+		// From AVCaptureDevice (AVCaptureDeviceType) Category
+		[Internal]
+		[iOS (10,0), NoMac]
+		[Export ("deviceType")]
+		NSString _DeviceType { get; }
+
+		[iOS (10, 0), NoMac]
+		[Wrap ("AVCaptureDeviceTypeExtensions.GetValue (_DeviceType)")]
+		AVCaptureDeviceType DeviceType { get; }
+
+		[Internal]
+		[iOS (10,0), NoMac]
+		[Static]
+		[Export ("defaultDeviceWithDeviceType:mediaType:position:")]
+		AVCaptureDevice _DefaultDeviceWithDeviceType (NSString deviceType, string mediaType, AVCaptureDevicePosition position);
+
+		[iOS (10,0), NoMac]
+		[Static]
+		[Wrap ("AVCaptureDevice._DefaultDeviceWithDeviceType (deviceType.GetConstant (), mediaType, position)")]
+		AVCaptureDevice GetDefaultDevice (AVCaptureDeviceType deviceType, string mediaType, AVCaptureDevicePosition position);
```

Comment:
What happens as `AVCaptureDeviceType` is a `[Flag]` enum ?
`GetConstant` does not deal with that - even if it did the underlying API would not accept an array